app/lib/actions/katello/repository/refresh_repository.rb

```diff
@@ -2,10 +2,13 @@ module Actions
   module Katello
     module Repository
       class RefreshRepository < Actions::Base
+        include Actions::Katello::PulpSelector
+
         def plan(repo, options = {})
           User.as_anonymous_admin do
             repo = ::Katello::Repository.find(repo.id)
-            plan_action(Pulp::Repository::Refresh, repo, :capsule_id => SmartProxy.default_capsule!.id, :dependency => options[:dependency])
+            plan_pulp_action([Actions::Pulp::Orchestration::Repository::RefreshIfNeeded],
+                             repo, SmartProxy.default_capsule, :dependency => options[:dependency])
```

Comment:
should default_capsule be defaul_capsule!

Reply:
yes!  without the typo